integration-intygsadmin/src/main/java/se/inera/intyg/minaintyg/integration/intygsadmin/BannerConverter.java

```diff
@@ -0,0 +1,27 @@
+package se.inera.intyg.minaintyg.integration.intygsadmin;
+
+import java.util.Arrays;
+import java.util.List;
+import org.springframework.stereotype.Component;
+import se.inera.intyg.minaintyg.integration.api.banner.model.Banner;
+import se.inera.intyg.minaintyg.integration.intygsadmin.client.dto.BannerDTO;
+
+@Component
+public class BannerConverter {
+
+  public List<Banner> convert(BannerDTO[] banners) {
```

Comment:
Since the API we are working against returns a collection, I dont see the reason to change this.

If the need of converting a single banner in the future should arise, I think we could handle that change by adding functionality to the BannerConverter that handles that scenario.